.github/workflows/go-syntax.yml

```
@@ -0,0 +1,26 @@
+name: Go Syntax Check
+
+on:
+  workflow_call:
+
+jobs:
+  syntax:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Go Format
+        run: |
+          GOFMT_OUTPUT=$(gofmt -l $(find . -path '*/vendor/*' -prune -o -name '*.go' -print))
+          if [ -n "$FMT_OUTPUT" ] ; then
```

Does this want to be:
```suggestion
          if [ -n "$GOFMT_OUTPUT" ] ; then
```

---

Good spot